test/views/incomeSources/cease/CeaseIncomeSourcesViewSpec.scala

```diff
@@ -100,6 +100,10 @@ class CeaseIncomeSourcesViewSpec extends TestSupport with ImplicitDateFormatter
       "all fields have value" in new Setup(false) {
         val table = document.getElementById("sole-trader-businesses-table")
 
+        println("ZZZZZZZ")
```

Comment:
If we can remove this, pls double check PR for warnings, as these are raise by PR-bot

Reply:
yes my bad